test_trello.py

```diff
@@ -46,9 +62,22 @@ def test_app():
     trello.delete_trello_board(board_id)
 
 @pytest.fixture(scope="module")
+# def driver():
+#     with webdriver.Firefox() as driver:
+#         yield driver
 def driver():
-    with webdriver.Firefox() as driver:
+    opts = webdriver.ChromeOptions()
```

Comment:
Now that you're using Chrome, you should delete the commented out Firefox section rather than just commenting it out. And update the README - tell the reader what they need to do to run the tests.

Also, delete geckodriver.log